src/GraphQL.Relay/GraphQL.Relay.csproj

```diff
@@ -1,9 +1,12 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
@Shane32  change to `<TargetFrameworks>netstandard2.1;netstandard2.0;net6</TargetFrameworks>` ?

Reply:
I would not unless we have an ifdef that requires it.  Nuget should automatically pull dependencies appropriately for the correct TFM regardless of what this package's target framework is.